lib/msal-common/src/index.ts

```diff
@@ -47,6 +47,7 @@ export {
 } from "./authority/AuthorityOptions";
 export { AuthorityFactory } from "./authority/AuthorityFactory";
 export { AuthorityType } from "./authority/AuthorityType";
+export { rawMetdataJSON, EndpointMetadata } from "./authority/AuthorityMetadata";
```

Comment:
Is `rawMetadataJSON` still used now that you've exported `EndpointMetadata`? If not then we can remove it from exports.